src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -317,9 +317,6 @@ public function get($id, $invalidBehavior = self::EXCEPTION_ON_INVALID_REFERENCE
 
                 return;
             }
-            if (isset($this->privates[$id])) {
```

Comment:
Same here, shouldn't it be moved above the check for shared services ? Otherwise, getting an already initialized private service will not trigger the deprecation